pkg/cmd/roachtest/operations/changefeeds/resume.go

```diff
@@ -0,0 +1,90 @@
+// Copyright 2025 The Cockroach Authors.
+//
+// Use of this software is governed by the CockroachDB Software License
+// included in the /LICENSE file.
+
+package changefeeds
+
+import (
+	"context"
+	gosql "database/sql"
+	"fmt"
+	"time"
+
+	"github.com/cockroachdb/cockroach/pkg/cmd/roachtest/cluster"
+	"github.com/cockroachdb/cockroach/pkg/cmd/roachtest/operation"
+	"github.com/cockroachdb/cockroach/pkg/cmd/roachtest/option"
+	"github.com/cockroachdb/cockroach/pkg/cmd/roachtest/registry"
+	"github.com/cockroachdb/cockroach/pkg/cmd/roachtest/roachtestflags"
+	"github.com/cockroachdb/cockroach/pkg/jobs"
+)
+
+// runResumeChangefeeds resumes all paused changefeed jobs by interacting with the cluster.
+func runResumeChangefeeds(
+	ctx context.Context, o operation.Operation, c cluster.Cluster,
+) registry.OperationCleanup {
+	// Establish a connection to the cluster.
+	conn := c.Conn(ctx, o.L(), 1, option.VirtualClusterName(roachtestflags.VirtualCluster))
+	defer func() {
+		_ = conn.Close()
+	}()
+
+	// Fetch all changefeed jobs with the status 'paused'.
+	allPausedCFJobs, err := fetchAllChangefeedJobsWithStatuses(ctx, conn, jobs.StatusPaused)
+	if err != nil {
+		o.Fatal(err)
+	}
+
+	if len(allPausedCFJobs) == 0 {
+		o.Status("no jobs are available to resume.")
+		return nil
+	}
+	// Resume the fetched changefeed jobs.
+	err = resumeChangefeeds(ctx, o, conn, allPausedCFJobs)
```

Comment:
Let's resume only 1 changefeed per operation. That will make it easier to isolate any issues that occur during testing.